### ISSUE TITLE
Create an Unity Asset Store Package

### ISSUE THREAD

**Issue by atteneder** [atteneder/glTFast#26]
For easier installation/usage.
Ideally an automated process.

**Comment by reubenkcoutinho** [comment]
Unity allows for [scoped pacakges](https://docs.unity3d.com/Manual/upm-scoped.html), so you could place them on your own registry

You could place the packages on Github and GitLab. They allow to create registries as well:
https://github.com/features/packages#pricing
https://docs.gitlab.com/ee/user/packages/package_registry/

**Comment by atteneder** [comment]
Hi Reuben,

That's a great idea. glTFast is already in openupm, another scoped registry. I haven't had the time to try it (and add it to the documentation) though.

Feel free to give it a try and tell me if it worked:

https://openupm.com

Regards,
Andi

**Comment by reubenkcoutinho** [comment]
I can confirm this work. Although users would need to know how to work with upm

**Comment by atteneder** [comment]
@reubenkcoutinho Thanks for testing! I'd assume it's fairly easy to set it up, wasn't it?

A clear plus to me is that OpenUPM doesn't produce costs (compared to GitHub packages) and I like the community spirit.

I'll test it soon myself and if there's no showstopper, I consider this issue resolved.

**Comment by reubenkcoutinho** [comment]
Hi Andi, 

It was fairly easy. I didn't have any issues getting it going.

I'm working on a package that should make it easier for users to install it. I'll share that as soon as I have it going.

Cheers,
Reuben

**Comment by reubenkcoutinho** [comment]
Hey Andi,

Check this out. I made an installer for glTFFast [here](https://github.com/reubenkcoutinho/glTFFastInstaller).

This should make it easy for anyone trying to install glTFFast.

This is my first time contributing to open source so I'm not sure I did anything incorrect (w.r.t license, mentions, etc)

**Comment by atteneder** [comment]
> This is my first time contributing to

Congratulations on your first open source project! 🥇

Making the install easier is indeed desirable. I'm not sure if adding another package is the best solution (since adding anther step kind of kills the purpose of saving another).

Nonetheless it's great input, Thanks!

Thinking out-of-the-box: Ideally users would just click a button on the project's site that opens Unity and installs everything (or starts an Install wizard)

**Comment by reubenkcoutinho** [comment]
Thanks for the feedback Andi. I've made this for developer who may not have come across using openupm-cli. Plus this also hopefully removes any dependencies for openupm-cli

**Comment by atteneder** [comment]
I took the time to take a look at the topic of "making installing easier".

OpenUPM offers an experimental *package installer* feature. It automatically generates an installer package:

[KtxUnity Installer](https://package-installer.glitch.me/v1/installer/OpenUPM/com.atteneder.ktx?registry=https%3A%2F%2Fpackage.openupm.com)

I took the link from the [OpenUPM site of KtxUnity](https://openupm.com/packages/com.atteneder.ktx/?subPage=versions#close) (to the upper right).

It's based on this API, which automatically generates an installer (similar to yours):
<https://package-installer.glitch.me>
On top of that it does not require GIT LFS.

It's not perfect, but so far the best I've seen. I'm confident Unity will make this even easier in the future.

Thanks for your work and for pointing me to this direction. I'll probably update the install instructions for my packages in favour of installer links rather soon.
